packages/playground/_stories/fiori/Wizard/Wizard.stories.ts

```diff
@@ -108,31 +108,188 @@ export const WizardOverview: StoryFn = () => html`
 		});
 
 		wizard${index}ToStep2.addEventListener("click", function () {
-			deselectAll(wizard${index}Wiz);
+			deselectAll${index}(wizard${index}Wiz);
 			setStep(wizard${index}Wiz, 1);
 			wizard${index}ToStep2.setAttribute("hidden", true);
 		});
 		wizard${index}ToStep3.addEventListener("click", function () {
-			deselectAll(wizard${index}Wiz);
+			deselectAll${index}(wizard${index}Wiz);
 			setStep(wizard${index}Wiz, 2);
 			wizard${index}ToStep3.setAttribute("hidden", true);
 		});
 		wizard${index}Finalize.addEventListener("click", function () {
 			alert("Done!");
 		});
 
-		function deselectAll(wizard) {
+		function deselectAll${index}(wizard) {
 			Array.from(wizard.children).forEach(function(step) {
 				step.selected = false;
 			});
 		}
 		function setStep(wizard, idx) {
-			var step = getStep(wizard, idx);
+			var step = getStep${index}(wizard, idx);
 			step.selected = true;
 			step.disabled = false;
 		}
-		function getStep(wizard, idx) {
+		function getStep${index}(wizard, idx) {
 			return Array.from(wizard.children)[idx];
 		}
 	</script>
 `;
+
+export const WizardPageMode: StoryFn = () => html`
+	<ui5-dialog id="dialog${++index}" stretch header-heading="Wizard">
+		<ui5-wizard id="wiz-${index}" renderMode="Page">
```

Comment:
`render-mode="Page"` should be set for an attribute

Reply:
Done.